### PR TITLE
Add info about JS debugging

### PR DESCRIPTION
**Pull request by nunofilipesf**

### CODE REVIEW

File: docs/pages/omnia3/Modeler/omnia3_modeler_developingbehaviours.md [hunk 1]
```diff
@@ -38,7 +39,7 @@ The project can have (depending of which behaviours are modeled) the following f
 * **Entity**: Contains the classes (one per entity) that support the execution of the operations modeled through [*Entity Behaviours*](omnia3_modeler_behaviours.html#2-types-of-behaviours).
   * To each entity will be generated a file which name will respect the rule _MyEntityName.**Operations.cs**_.
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Modeler/omnia3_modeler_developingbehaviours.md]: A file will be generated for each entity and its name will respect the rule _MyEntityName.**Operations.cs**_.

File: docs/pages/omnia3/Modeler/omnia3_modeler_developingbehaviours.md [hunk 2]
```diff
@@ -38,7 +39,7 @@ The project can have (depending of which behaviours are modeled) the following f
...
 
-## 5. Debugging
+## 5. C# Debugging
 It's possilbe to debug all the C# behaviours, using the _Visual Studio_ and all of its debugging features.
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Modeler/omnia3_modeler_developingbehaviours.md]: All C# behaviours can be debugged using _Visual Studio_ and all of its debugging features.

File: docs/pages/omnia3/Modeler/omnia3_modeler_developingbehaviours.md (hunk 1 above)
Comment (gsferreira) [reply docs/pages/omnia3/Modeler/omnia3_modeler_developingbehaviours.md]: Done

File: docs/pages/omnia3/Modeler/omnia3_modeler_developingbehaviours.md (hunk 2 above)
Comment (gsferreira) [reply docs/pages/omnia3/Modeler/omnia3_modeler_developingbehaviours.md]: Done. Changed it a bit. Please read it again

**Pull request merged**